docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+This demo will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH denominated in USD. For example, if the user wants to withdraw $10,000 USD of wETH, and the ETH/USD exchange rate is $2,000, they would withdraw 5 wETH.
+
+* The user links the `OptimisticDepositBox` with one of the price identifiers enabled on the DVM.
+
+* The user deposits wETH into the `OptimisticDepositBox` and register it with the `ETH/USD` price identifier.
+
+* The user can now withdraw a USD-denominated amount of wETH from their `DepositBox` via smart contract calls, with the Optimistic Oracle enabling optimistic on-chain pricing.
+
+In this example, the user would not have been able to transfer USD-denominated amounts of wETH without referencing an off-chain `ETH/USD` price feed. The Optimistic Oracle therefore enables the user to "pull" a reference price.
+
+Unlike price requests to the DVM, a price request to the Optimistic Oracle can be resolved within a specified liveness window if there are no disputes, which can be significantly shorter than the DVM voting period. The price requestor is not currently required to pay fees to the DVM.
+
+The requestor can offer a reward for the proposer who responds to a price request, but the reward value is set to `0` in this example.
+
+The price proposer posts a bond along with their price, which will be refunded if the price is not disputed, or if a dispute is resolved in the proposer's favor. Otherwise, this bond is used to pay the final fee to the DVM and pay a reward to a successful disputer.
```

Comment:
No bond in this example, right? Think this should be mentioned - could see people wondering if they have to "pay"

Reply:
The bond is collected automatically in the `proposePriceFor` function called by the proposer, but it's only the `finalFee` in this case.

Reply:
I can make this more explicit.